docs: Update admin guide with instructions on creating a new GitHub repo from a local one. (#102)
seedcase-project/team: entries/admin.qmd

``` bash
gh repo edit --delete-branch-on-merge=true --enable-wiki=false --enable-discussions=false --enable-auto-merge=true --allow-update-branch
```

## Creating a GitHub repo from a local one

> If a local repository exists and a GitHub repository needs to be created for that repository, run this code using the Terminal while *inside* the local Git repository folder (the root folder of the Git repository), that does the following actions:

Reviewer: ```suggestion
If a local repository exists and a GitHub repository needs to be created for that repository, run this command in the Terminal while *inside* the local Git repository folder (the root folder of the Git repository), that does the following actions:
```

- Creates a repo in the `seedcase-project` organization called `REPO` (rename to the folder name of the local repo).

> - Sets the repo to public (not private) via the `--public`.

Reviewer: ```suggestion
- Sets the repo to public (not private) via the `--public` flag.
```

> - Use the root folder as the local Git repository with `--source=.`.

Reviewer: ```suggestion
- Uses the root folder as the local Git repository with `--source=.`.
```

> - Include a description of the newly created repository with `--description=""`.

Reviewer: ```suggestion
- Include a description of the newly created repository with `--description="WRITE DESCRIPTION"` (write the description of what's in the repo instead of *WRITE DESCRIPTION*). 
```

> - Include a description of the newly created repository with `--description=""`.

Reviewer: ```suggestion
- Includes a description of the newly created repository with `--description=""`.
```

> - Push the local repo to the new repo with `--push`.

Reviewer: ```suggestion
- Pushes the local repo to the new repo with `--push`.
```

> - Disable the wiki with `--disable-wiki`.

Reviewer: ```suggestion
- Disables the wiki with `--disable-wiki`.
```

> - Set the URL for the homepage with `--homepage`, changing the `REPO` with the name of the root folder of the Git repo.

Reviewer: ```suggestion
- Sets the URL for the homepage with `--homepage` (change `REPO` in the link following this flag with the name of the root folder of the Git repo).
```
